cluster/manifests/stackset-controller/deployment.yaml

```diff
@@ -30,12 +30,15 @@ spec:
       serviceAccountName: stackset-controller
       containers:
       - name: stackset-controller
-        image: "container-registry.zalando.net/teapot/stackset-controller:{{ $version }}"
+        image: "container-registry-test.zalando.net/teapot/stackset-controller:pr-520-36"
```

Comment:
Let's keep `:{{ $version }}` (We configure it on the first line)

Reply:
This is a temporary image, this line is never touching dev. We get back to the proper format once the the actual image is ready.

Reply:
But why did you change the `$version` variable on line 1 then?

Reply:
I did that after because I thought I may want to specify the test version on the labels. No obvious reason, just a "maybe it'd be useful".